doc/content/resources/neams-workbench.md

```diff
@@ -0,0 +1,162 @@
+
+# Cloning the VTB GitHub repository
+
+Before running the NEAMS Workbench GUI on the INL HPC platform, it is recommended to clone the VTB GitHub repository to your home directory on Sawtooth to access the input files used in the subsequent sections. This can be achieved from the [HPC OnDemand website](https://hpcondemand.inl.gov/webauthentication) by starting a Sawtooth terminal as shown in [fig:hpc_on_demand_sawtooth_terminal].
+
+!media neams_workbench/hpc_on_demand_sawtooth_terminal.png style=width:80% id=fig:hpc_on_demand_sawtooth_terminal caption=Accessing Sawtooth terminal on the INL HPC OnDemand website.
+
+The Sawtooth terminal should immediately start as seen in [fig:sawtooth_terminal]:
+
+!media neams_workbench/sawtooth_terminal.png style=width:80% id=fig:sawtooth_terminal caption=Sawtooth terminal on the INL HPC OnDemand website.
+
+Once the terminal is opened, the VTB GitHub repository can be cloned. The first step is to load the `gitlfs` module and to install `Git LFS` locally. [Git-LFS](https://git-lfs.github.com/) is needed to clone large size files (mesh, database, ...) present in the VTB GitHub repository:
+
+```console
+module load gitlfs; git lfs install
+```
+
+If successful, the terminal should show `Git LFS initialized.` Once git-lfs is enabled, copy the successive command lines to your Sawtooth terminal and hit enter to clone the VTB repository:
+
+```console
+git clone https://github.com/idaholab/virtual_test_bed.git virtual_test_bed
+```
+
+The output of the above command line is as follows:
+
+```shell
+Cloning into 'virtual_test_bed'...
+remote: Enumerating objects: 4524, done.
+remote: Counting objects: 100% (885/885), done.
+remote: Compressing objects: 100% (366/366), done.
+remote: Total 4524 (delta 534), reused 719 (delta 502), pack-reused 3639
+Receiving objects: 100% (4524/4524), 112.31 MiB | 39.01 MiB/s, done.
+Resolving deltas: 100% (2487/2487), done.
+Downloading htgr/htr10/data/mesh/htr-10-critical-a-rev6.e (2.7 MB)
+Downloading htgr/htr10/data/mesh/htr-10-full-a-rev3.e (3.5 MB)
+Downloading htgr/htr10/data/sph/htr-10-Diff-SPH.xml (3.0 MB)
+Downloading htgr/htr10/data/xs/htr-10-XS.xml (4.9 MB)
+Downloading htgr/htr10/steady/gold/out-HTR-10-full-Diff-SPH-ARO.e (7.4 MB)
+Downloading htgr/htr10/steady/gold/out_HTR-10_critical.e (7.9 MB)
+Downloading htgr/pbmr400/shared/oecd_pbmr400_tabulated_xs.xml (212 MB)
+Downloading mrad/mesh/mrad_mesh.e (62 MB)
+Downloading pbfhr/reflector/conduction/conduction.fld (247 MB)
+Downloading pbfhr/reflector/meshes/fluid.re2 (60 MB)
+Downloading pbfhr/reflector/meshes/solid.e (62 MB)
+Checking out files: 100% (370/370), done.
+```
+
+The content of the `virtual_test_bed` directory can be accessed with the `ls` command:
+
+```console
+ls virtual_test_bed/
+apps  COPYRIGHT  doc  htgr  LICENSE  mrad  msr  pbfhr  README.md  scripts  sfr
+```
+
+All VTB examples are run with the *BlueCrab* application (see Section 5 for further details); users are referred to Section 4 for details.
```

Comment:
```suggestion
All VTB examples can be run with the *BlueCrab* application (see Section 5 for further details); users are referred to Section 4 for details.
```

Does workbench not support all the others?

Reply:
It does but I think the idea is to have BlueCrab as the main executable to run all applications. It was only tested with BlueCrab on Sawtooth for now.